dice_test.go

```diff
@@ -8,7 +8,7 @@ import (
 
 func TestFlipCoin(t *testing.T) {
 	for i := 1; i < 10; i++ {
-		result := flipCoin()
+		result := flipCoin("coin")
```

Comment:
Can we write some tests for the new functionality?
Many coins, too many coins, negative number of coins etc

Reply:
Added some tests.